jbi/queue.py

```diff
@@ -73,8 +73,9 @@ async def get(self, bug_id: int) -> list[QueueItem]:
     async def get_all(self) -> dict[int, list[QueueItem]]:
         pass
 
+    @property
     @abstractmethod
-    async def size(self) -> int:
+    def size(self) -> int:
```

Comment:
Are you sure about this one?
I was thinking that if we end up implementing the queue in a db/redis or whatever, we would need it to be async... 
Maybe YAGNI, just checking :)